test/suite/Annotations.test.php

```diff
@@ -580,17 +587,75 @@ protected function testGetAnnotationsFromAnInterface()
 
     protected function testGetAnnotationsFromTrait()
     {
-        if (version_compare(PHP_VERSION, '5.4.0', '<')) {
```

Comment:
We require PHP 5.3+ and same test suite is executed on each PHP version on Travis CI. This way if we try to use PHP 5.4 code (e.g. traits) unconditionally in test suite, then this would result in Fatal Error on smaller PHP versions. That's why we have that check in there to make test pass on PHP versions, where traits are not supported.